### DISCUSSION TITLE
Future of Blazor WebAssembly

### DISCUSSION THREAD

**Discussion by JuanIrigoyen** [dotnet/aspnetcore#50652] [General]
Good morning, we have already developed several applications in Blazor and the truth is that we are delighted despite a serious problem that prevents us from being agile related to the hot reload that most of the time does not work.
Regardless of this, there is a discussion with other people working in international companies who comment that the bet by Microsoft with Blazor is not real, that they have a very small team working on its evolution and that it is easy that it does not have a clearly defined future and ends up disappearing. 

In my company the bet for Blazor is more and more solid, but I would like that since you are the ones who evolve this project you tell me first hand your opinion since I have 'suffered' other Microsoft projects that have ended up 'failing', it is the case of Silverlight and other projects.

It is curious that some parts of the Windows 11 presentation layer and other applications of its environment use React, Electron and other alternatives, but not Blazor.

In my opinion the WebAssembly standard is a clear bet for the future and I hope Blazor continues to evolve because our company will rely more and more on this technology and I would not like to find myself with a failed project for what this may mean. 

I would like to have more information about Microsoft's bet on this technology and its evolution, can you give me your opinion about it?

**Comment by DavidThielen** [comment]
I can't speak for Microsoft. But pretty much every ASP.NET dev team I know, for new projects, is going with Blazor. And it seems to be 2/3 web assembly, 1/3 server. (I prefer Blazor server for a database centric app.)
